plugins/push/frontend/public/stylesheets/main.css

```diff
@@ -987,7 +1015,7 @@ table.d-table .nothing { opacity: 0.3; }
 
 .comp-push input[type="number"]::-webkit-inner-spin-button,
 .comp-push input[type="number"]::-webkit-outer-spin-button {
-    -webkit-appearance:none;
```

Comment:
Ok to be honest, I couldn't find a solution to return back input-number to browser defaults. I asked on stack-overflow, but nobody has answered it yet. https://stackoverflow.com/questions/45230816/browser-default-value-for-input-number-arrows